.github/workflows/build.yml

```diff
@@ -7,10 +7,10 @@ on:
 name: Build
 
 jobs:
-  host_tests:
+  macos_and_windows:
     strategy:
       matrix:
-        os: [macos-latest, windows-2019]
+        os: [macos-latest, windows-2019, macOS]
```

Comment:
What is the new macOS param doing?

Reply:
ah I see trying old macos, but is that just to try or a permanent part of testing?